### PR TITLE
Create tizen-9-0-m1.md

### PR DESCRIPTION
**Pull request by seokwon-jung**
### Change Description ###

Describe your changes here.

Create tizen-9-0-m1.md

### Bugs Fixed ###

Provide links to bugs here

For instance,

 - Issue #265

### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1120

### CODE REVIEW

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Thank you for posting the release note.

However, it seems like you have missed out on adding the file to the `toc_all.md` file in the Platform folder. Please add it to the TOC so that it appears properly in the LNB.

File: docs/platform/release-notes/tizen-9-0-m1.md [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+  - Kernel for Raspberry Pi 4 has been upgraded to version 6.6.17.
+  - Raspberry Pi 5 has been supported with the ARM64 Raspberry Pi 4 BOOT image
+    - Known issue: Raspberry Pi 5 board should have EEPROM version 2013-12-06.
+
+- System and resource management
+  - The ISU of resourced package has been added.
+
+- Device and sensor management
+  - The priority-based sound feedback playback has been added and the getter of supported theme ID list.
+  - The key mapping between linux kernel input key code and button has been supported when adding new hardware button.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-9-0-m1.md:32]:
```suggestion
  - The key mapping between Linux kernel input key code and button has been supported when adding new hardware button.
```

File: docs/platform/release-notes/tizen-9-0-m1.md [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+- OS upgrade
+  - The RPK format of ISU package has been added. It is possible to install/uninstall ISU packages via pkgcmd tool.
+  - The update-control plugin has been supported in order to separate the vendor/product dependent code.
+
+- kmod
+  - Version 30 has been upgaded to version 31.
+- f2fs-tools
+  - Version 1.14.0 has been upgaded to version 1.16.0.
+- libdrm
+  - Version 2.4.114 has been upgaded to version 2.4.118.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-9-0-m1.md:44]:
```suggestion
- kmod
  - Version 30 has been upgraded to version 31.
- f2fs-tools
  - Version 1.14.0 has been upgraded to version 1.16.0.
- libdrm
  - Version 2.4.114 has been upgraded to version 2.4.118.
```

File: docs/platform/release-notes/tizen-9-0-m1.md [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+  - KDBus has been supported on the upgraded glib 2.78.4.
+
+
+#### Fixes
+- sessiond
+ - The dependency with libsessiond has been removed to reduce the memory usage when it is not used.
+- storaged
+ - Tizen.System.Feedback issue that does not detect 'unmountable event' has been fixed.
+- sensord
+ - The exception handling code when listener connection is failed has beed added.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-9-0-m1.md:57]:
```suggestion
 - The exception handling code when listener connection is failed has been added.
```

File: docs/platform/release-notes/tizen-9-0-m1.md [hunk 4]
```diff
@@ -0,0 +1,543 @@
...
+  - The API to get the maximum buffer size of L2CAP socket has been added.
+- Open source
+  - Curl has been upgraded to version 8.5.0.
+
+#### Fixes
+
+- Network
+  - The logic to parse device information in multiple device environments has been fixed.
+  - D-Bus signal handling code for Wi-Fi has been fixed.
+  - The issues in the code for storing and loading wifi configuration have been fixed.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-9-0-m1.md:404]:
```suggestion
  - The issues in the code for storing and loading Wi-Fi configuration have been fixed.
```

File: docs/platform/release-notes/tizen-9-0-m1.md [hunk 5]
```diff
@@ -0,0 +1,543 @@
...
+- Push Client
+  - Potential defects have been fixed.
+
+### Web framework
+
+#### New and changed features
+
+- WebGPU has been supported.
+- Sharescreen API has been supported.
+- Picture in Picture(Document, Media Session API) has been supported.
```
Comment (safir-srbd) [inline docs/platform/release-notes/tizen-9-0-m1.md:442]:
```suggestion
- Picture in Picture (Document, Media Session API) has been supported.
```

**Review by safir-srbd — COMMENTED** [review]
@seokwon-jung Thank you for adding the doc to the toc. It is appearing correctly on the LNB now. I have suggested a few changes.

**IMPORTANT:** Please make a change to the `tizen-8-0-m1.md` file and change the source code branch from 'tizen' to 'tizen_8.0'.

**Comment by seokwon-jung** [comment]
> @seokwon-jung Thank you for adding the doc to the toc. It is appearing correctly on the LNB now. I have suggested a few changes.
>  udpated/fixed ,thanks 

> **IMPORTANT:** Please make a change to the `tizen-8-0-m1.md` file and change the source code branch from 'tizen' to 'tizen_8.0'.
> sorry, I don't understand what you mean..?

**Review by safir-srbd — APPROVED** [review]
@seokwon-jung Thank you for making the suggested changes. Everything seems to be in order now. I will approve this PR and merge it on the date of release.

**Pull request merged**